torchtitan/experiments/flux/tests/unit_tests/test_flux_dataloader.py

```diff
@@ -97,3 +99,63 @@ def _build_dataloader(
             tokenizer=None,
             infinite=True,
         )
+
+    def _test_flux_dataloader_checkpointing(self, dataset_name):
+        for world_size in [2, 4]:
+            for rank in range(world_size):
+                batch_size = 4
+
+                path = "torchtitan.experiments.flux.job_config"
+                config_manager = ConfigManager()
+                config = config_manager.parse_args(
+                    [
+                        f"--experimental.custom_args_module={path}",
+                        # Profiling options
```

Comment:
If these options are not needed, pls delete them, I forgot to clean them in `_test_flux_dataloader`